CHANGES.md

```diff
@@ -1,5 +1,30 @@
 # Changes
 
+## edge-21.6.2
+
+This release fixes a problem with the HTTP body buffering that was added
+to support gRPC retries. Now, Only requests with a retry configuration
+are buffered (and only when their bodies are less than 64KB).
+
+Additionally, An issue with the outbound ingress-mode proxy where forwarded
+HTTP traffic could fail to detect when the target pod was deleted, causing
```

Comment:
The actor is confused here. I don't think _forwarded HTTP traffic_ is responsible for detecting anything. I presume it's whichever thing is _forwarding_ http traffic that's supposed to detect the pod is deleted.